processing/src/main/java/org/apache/druid/segment/filter/Filters.java

```diff
@@ -26,11 +26,14 @@
 import org.apache.druid.query.DefaultBitmapResultFactory;
 import org.apache.druid.query.Query;
 import org.apache.druid.query.QueryContexts;
+import org.apache.druid.query.extraction.ExtractionFn;
+import org.apache.druid.query.filter.BooleanFilter;
 import org.apache.druid.query.filter.ColumnIndexSelector;
 import org.apache.druid.query.filter.DimFilter;
 import org.apache.druid.query.filter.DruidPredicateFactory;
 import org.apache.druid.query.filter.Filter;
 import org.apache.druid.query.filter.FilterTuning;
+import org.apache.druid.query.filter.SelectorDimFilter;
```

Comment:
This should not be imported, we are not using SelectorDimFilter in the changed code